examples/eventDriven/src/customSpec.yml

```
@@ -0,0 +1,32 @@
+info:
+  title: User Event API
+  version: 1.0.0
+description: ' User Event API Specification'
```

we need this space here?

```suggestion
description: 'User Event API Specification'
```